mojave/items/guns/weapons/energy/energy_pistol.dm

```diff
@@ -24,6 +24,7 @@
 	base_icon_state = "advlas_pistol"
 	inhand_icon_state = "advlas_pistol"
 	fire_sound = 'mojave/sound/ms13weapons/gunsounds/laspistol/las_pistol_2.ogg'
+	cell_type = /obj/item/stock_parts/cell/ms13/mfc
```

Comment:
You'll need to make a new ammo_type for the adv. las pistol that has proper energy usage to put it at 20 shots with the amount of energy that the MFC has. Laser pistols are balanced around/intended to have 20 shots.